rsconf/component/sirepo_test_http.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+u"""Timer to run sirepo test_http
+
+:copyright: Copyright (c) 2018 Bivio Software, Inc.  All Rights Reserved.
+:license: http://www.apache.org/licenses/LICENSE-2.0.html
+"""
+from __future__ import absolute_import, division, print_function
+from pykern import pkcollections
+from pykern.pkdebug import pkdp
+from rsconf import component
+
+
+class T(component.T):
+    def internal_build(self):
+        from rsconf import systemd
+        from rsconf.component import docker_registry
+
+        self.buildt.require_component('sirepo')
```

Comment:
You don't want this. This will start sirepo on the test machine

Reply:
https://github.com/radiasoft/rsconf/pull/117/commits/2167e9eafbbb857823fe3d83e6ca395e3f22b0ce